qcengine/programs/psi4.py

```diff
@@ -164,7 +164,7 @@ def compute(self, input_model: "AtomicInput", config: "TaskConfig") -> "AtomicRe
                         else:
                             output_data["extras"]["qcvars"] = local_qcvars
 
-                    if output_data["success"] is False:
+                    if output_data.get("success", False) is False:
```

Comment:
To be safe you may want to protect the below dict getters in a similar fashion `error_message = output_data.get("error", "Unknown error, error message is not found")` or whatnot.

Reply:
Ah thanks! Will add this now.